Conceptpower+Spring/src/main/java/edu/asu/conceptpower/root/DatabaseClient.java

```diff
@@ -296,6 +302,84 @@ public boolean match(ConceptList dict) {
      * lang.String)
      */
     @Override
+    public List<ConceptEntry> getAllEntriesFromList(String listname, int page, int pageSize, final String sortBy,
+            final int sortDirection) {
+        ConceptEntry entry = new ConceptEntry();
+        entry.setConceptList(listname);
+
+        Query wordNetQuery = wordnetCacheClient.query();
```

Comment:
This method was fetching the concept entries from both wordnetcache client and dictionary cache client and grouped it together. So I followed the same approach of fetching the concept entries from wordnet cache and dictionary cache. Please let me know if I need to remove wordnet cache client here.